src/channel.h

```diff
@@ -98,7 +98,7 @@ class CChannel : public QObject
     bool GetAddress ( CHostAddress& RetAddr );
     const CHostAddress& GetAddress() const { return InetAddr; }
 
-    void ResetInfo() { ChannelInfo = CChannelCoreInfo(); } // reset does not emit a message
+    void ResetInfo() { bIsIdentified = false; ChannelInfo = CChannelCoreInfo(); } // reset does not emit a message
```

Comment:
Where is this called from?  This change makes it do more than its name declares.  It now resets info and stops audio.

Reply:
No, strictly speaking, it _is_ just resetting the info, including the `bIsIdentified` flag, which is just conveniently indicating there is no info. The fact that audio stops is due to decisions elsewhere in the code.

It is called in the constructor for `CChannel` and also in `CServer::PutAudioData` when it is about to use or re-use a `CChannel`.